static/js/compose_actions.js

```diff
@@ -292,10 +292,22 @@ export function start(msg_type, opts) {
         return;
     }
     notifications.clear_compose_notifications();
-    expand_compose_box();
 
     opts = fill_in_opts_from_current_narrowed_view(msg_type, opts);
 
+    if (
```

Comment:
A comment here explaining that we close the compose box when narrowing to a deactivated user, else we expand it, would be nice